packages/showcases/css/.storybook/preview-head.html

```diff
@@ -36,6 +36,13 @@
   }
   .block {
     margin-bottom: 1rem;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .block > * {
+    margin: 0 20px;
```

Comment:
Could be nice to not apply this for first and last children because we already have padding.

![Capture d’écran 2021-11-15 à 02 06 07](https://user-images.githubusercontent.com/9600228/141707016-5653f9b8-3a92-4e66-a402-cb91c02c3f5e.png)

As you can see, it's not aligned.